src/app/pages/credentials/certificates-dash/certificate-edit/certificate-edit.component.ts

```diff
@@ -54,19 +52,25 @@ export class CertificateEditComponent implements OnInit {
     private errorHandler: FormErrorHandlerService,
     private matDialog: MatDialog,
     @Inject(SLIDE_IN_DATA) private slideInData: SlideInDataCertificateEdit,
-  ) {}
+  ) {
+    this.form.addControl('name', new FormControl(''));
+  }
 
   get isCsr(): boolean {
     return this.certificate?.cert_type_CSR;
   }
 
   ngOnInit(): void {
     this.setCertificate();
+
+    if (this.certificate?.acme) {
```

Comment:
Lets make it a well named method like `setCertificate`

Reply:
okay, gonna refactor it.